tests/e2e/tests/build/bundle-size.ts

```diff
@@ -0,0 +1,34 @@
+import {exec} from '../../utils/process';
+import {uploadBundleJsFileSize, expectGlobFileSizeToBeUnder} from '../../utils/fs';
+import {ng} from '../../utils/process';
+
+const PROJECT_ID = 'angular-payload-size';
+
+export default async function() {
+  await ng('build', '--prod');
+
+  // Expect size of the main bundle AND the whole app to be within 1% of an "Hello World" project.
+  await expectGlobFileSizeToBeUnder('dist/main.*.js', 152200 * 1.01);
+  await expectGlobFileSizeToBeUnder('dist/*', 232000 * 1.01);
+
+  // Upload payload size to firebase
+  const payloadData = await uploadBundleJsFileSize('dist/*.bundle.js');
+
+  // Add timestamp and commit message
+  payloadData['timestamp'] = Math.floor(Date.now()/1000);
+  payloadData['message'] = process.env['TRAVIS_COMMIT_MESSAGE']
+      .replace(/\\/g, '\\').replace(/"/g, '\"').replace(/'/g, "\'");
```

Comment:
Those replacements sound wrong. Seems that if you want to replace `\` with `\\` you could use `JSON.stringify(process.env['TRAVIS_COMMIT_MESSAGE'])`.